docs/SPIR-V.rst

```diff
@@ -1631,3 +1639,77 @@ element is the height, and the third is the elements.
 ++++++++++++++++++++++++++++++++++++++++
 The ``OpImageQuerySize`` instruction is used to get a uint3. The first element is the width, the second
 element is the height, and the third element is the depth.
+
+
+HLSL Hull Shaders
+=================
+
+Hull shaders translate to Tessellation Control Shaders (TCS) in Vulkan.
+This section describes how Hull shaders are translated to SPIR-V for Vulkan.
+
+Hull Entry Point Attributes
+---------------------------
+The following HLSL attributes are assigned to the main entry point of hull shaders
+and are translated to SPIR-V execution modes according to the table below:
+
++-------------------------+---------------------+--------------------------+
+| HLSL Attribute          |   value             | SPIR-V Execution Mode    |
++=========================+=====================+==========================+
+|                         | ``quad``            | ``Quads``                |
+|                         +---------------------+--------------------------+
+|    ``domain``           | ``tri``             | ``Triangles``            |
+|                         +---------------------+--------------------------+
+|                         | ``isoline``         | ``Isoline``              |
++-------------------------+---------------------+--------------------------+
+|                         | ``integer``         | ``SpacingEqual``         |
+|                         +---------------------+--------------------------+
+|                         | ``fractional_even`` | ``SpacingFractionalEven``|
+|    ``partitioning``     +---------------------+--------------------------+
+|                         | ``fractional_odd``  | ``SpacingFractionalOdd`` |
+|                         +---------------------+--------------------------+
+|                         | ``pow2``            |           N/A            |
++-------------------------+---------------------+--------------------------+
+|                         | ``point``           | ``PointMode``            |
+|                         +---------------------+--------------------------+
+|                         | ``line``            |           N/A            |
+|  ``outputtopology``     +---------------------+--------------------------+
+|                         | ``triangle_cw``     | ``VertexOrderCw``        |
+|                         +---------------------+--------------------------+
+|                         | ``triangle_ccw``    | ``VertexOrderCcw``       |
++-------------------------+---------------------+--------------------------+
+|``outputcontrolpoints``  | ``n``               | ``OutputVertices n``     |
++-------------------------+---------------------+--------------------------+
+
+The last attribute is `patchconstfunc` which does not have a direct equivalent
```

Comment:
The patchconstantfunc attribute does not...

Double back-tick for rendering as code.